### DISCUSSION TITLE
how to force format

### DISCUSSION THREAD

**Discussion by Paul74** [jely2002/youtube-dl-gui#443] [Q&A] [answered]
Hi, for a YT video wich has 5.1 audio the downloader only gets the stereo format (251) even if I set "best audio". The 5.1 should be the 258 code format. Is there a way to force it? The video is https://youtu.be/OaJ5tfZksbk
BTW with another YT video with 5.1 the program correctly downloaded the 6 channel stream with just the best audio setting, so I don't know...
Anyway I'd like to know how to set the interface or the custom filename format to force to get the 5.1 audio stream. Thank you.

**Comment by Paul74** [comment, marked as answer]
actually the youtube-dl list of formats states that there is no more then 2ch streams...how strange...the video claims to be 5.1.

**Comment by jely2002** [comment]
Its now possible to force it using the "track" selection, there you can selecht the 2ch track.
